### PR TITLE
docs: fix broken quick start links

### PR DESCRIPTION
**Pull request by tomleah**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description

Fixed a few links pointing to the quick start guide

### CODE REVIEW

**Comment by Kamran64** [comment]
LGTM but I'm noticing the side bar isn't highlighting 'Quick Start' when following the link. Not sure if this is an issue on your end or connected to #106

File: content/docs/getting-started/installation.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ brew install appsody/appsody/appsody
 
 This command creates a brew tap for the `appsody/appsody` repo and installs the `appsody` formula.
 
-That's all there is to it! You can now follow our [Quick Start](#quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
+That's all there is to it! You can now follow our [Quick Start](/docs/getting-started/quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: add `.md` on the end here

File: content/docs/getting-started/installation.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ sudo apt install -f <path>/appsody_v.r.m_amd64.deb
 ```
 Where `<path>` is the fully qualified path to the package.
 
-That's all there is to it! You can now follow our [Quick Start](#docs/getting-started/quick-start.md) instructions to create your first containerized development environment with a running *Hello World!* application.
+That's all there is to it! You can now follow our [Quick Start](/docs/getting-started/quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: same here

File: content/docs/getting-started/installation.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ sudo yum install <path>/appsody-v.r.m-1.x86_64.rpm
 
 Where `<path>` is the fully qualified path to the package.
 
-That's all there is to it! You can now follow our [Quick Start](#docs/getting-started/quick-start.md) instructions to create your first containerized development environment with a running *Hello World!* application.
+That's all there is to it! You can now follow our [Quick Start](/docs/getting-started/quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: same here

File: content/docs/getting-started/installation.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ Follow these steps:
 appsody-setup.bat
 ```
 
-That's all there is to it! You can now follow our [Quick Start](#docs/getting-started/quick-start.md) instructions to create your first containerized development environment with a running *Hello World!* application.
+That's all there is to it! You can now follow our [Quick Start](/docs/getting-started/quick-start) instructions to create your first containerized development environment with a running *Hello World!* application.
```
Comment (mhamwala) [inline content/docs/getting-started/installation.md]: same here

**Comment by tomleah** [comment]
As we expect github links **not** to work, adding `.md` doesn't really make a difference, but i've added for consistency

**Comment by tomleah** [comment]
@Kamran64 i'm not able to replicate the issue you are having :/ @mhamwala are you able to verify this?

**Pull request merged**